modules/installation-user-infra-machines-pxe.adoc

```diff
@@ -92,7 +116,12 @@ installation, do not delete these files.
 . Configure the network boot infrastructure so that the machines boot from their
 local disks after {op-system} is installed on them.
 
-. Configure PXE or iPXE installation for the {op-system} images and begin the installation.
+ifndef::only-pxe[]
+. Configure PXE or iPXE installation for the {op-system} images.
```

Comment:
I added "and begin the installation" as part of [PR 32618](https://github.com/openshift/openshift-docs/pull/32618/files#diff-3e2e52939307b0dbc8d700d3f8ddb0b4ef35b3271f5a8b4fee21ada94b03a176R95) to clarify that during this step the RHCOS installations begin. Without stating this, the step goes from configuring PXE to the next step which is to monitor the progress of the installation, without stating that the installation should be started.

You could add "and begin the installation" back to the end of this step in both conditionals, so that the reader is clear that the PXE installation starts here.